core/domain/wipeout_service.py

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
I didn't see this line in other files. Is the lint check throwing any error?

Reply:
Unwanted change. Removed